docs/faq.md

```diff
@@ -266,3 +266,24 @@ Alternatively, perform a runtime check for the EarlGrey class:
 // Swift
 public static let envEarlGrey:Bool = NSClassFromString("EarlGreyImpl") != nil
 ```
+
+**How do I find off screen elements?**
+
+EarlGrey requires elements to be visible (in the UI hierarchy) to perform automation.
+Just as a user would, scroll elements into view before interacting with them.
+
+```swift
+// Swift
+EarlGrey.selectElementWithMatcher(matcher)
```

Comment:
One more suggestion:
matcher should contain grey_interactable or grey_sufficientlyVisible if the intention is to eventually interact with this element. Otherwise, the search action will stop as soon as it finds an element that satisfies `matcher` and try to perform an action which may have grey_sufficientlyVisible as its constraint and fail because element isn't visible (but it exists in the UI hierarchy)